src/app/pages/datasets/components/dataset-form/dataset-form.component.ts

```diff
@@ -1063,6 +1067,22 @@ export class DatasetFormComponent implements FormConfiguration {
       this.pk = this.parent;
       this.isNew = true;
       entityForm.formGroup.controls['parent'].setValue(this.parent);
+      entityForm.formGroup.controls['name'].setValidators(datasetNameTooLong(this.parent));
```

Comment:
let's separate if () block from others with empty line

Reply:
I'm not sure I understand the issue. Can you elaborate please?

Reply:
@RehanY147 

const ...

if ()

So empty line between variables and if statements